exporter/datadogexporter/testtool.go

```diff
@@ -0,0 +1,156 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build ignore
+// +build ignore
+
+// TODO: Only supports Metrics Sum & Gauge. To implement more, update the "readMetrics" function
+// from internal/testutil/stdout.go.
+//
+// For additional telemetry, update ReadLoggingExporter and the endpoints in this file.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/kr/pretty"
+	"github.com/mitchellh/mapstructure"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v2"
+	"gopkg.in/zorkian/go-datadog-api.v2"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/testutil"
+)
+
+const pathSeries = "/api/v1/series"
+
+func main() {
```

Comment:
We usually use cobra for commands.

Reply:
If there is interest on having this as a common utility, I think we should put it under `./cmd`, it should be useful to all, not only Datadog

Reply:
If we want to make it available to everyone, I'd suggest making the [otlptext](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter/internal/otlptext) package exporter instead, and adding this as an `Unmarshal` method.